src/Apps/Auction/Components/BidForm.tsx

```diff
@@ -137,6 +152,48 @@ export const BidForm: React.FC<Props> = ({
                   {showPricingTransparency && <PricingTransparency />}
                 </Flex>
 
+                {requiresPaymentInformation && (
+                  <Box>
+                    <Separator mb={3} />
+
+                    <Serif size="4" color="black100">
```

Comment:
this block is identical to:

https://github.com/artsy/reaction/blob/6222a8ad9ee209f5ec4974b2d40d51d35de2cb2c/src/Apps/Auction/Components/RegistrationForm.tsx#L224

– thoughts on extracting some form of shared component? Particularly with content, we would they blocks to change together.

Reply:
Yes, there is a large opportunity for refactoring. @bhoggard are I looked at some other stuff like GraphQL calls, small functions, validation schemas, and even consolidating the `RegistrationForm` and `BidForm` , but it seemed like a lot of refactoring.

We might want to loop @dblandin and @erikdstock in to get it done well, but I don't think that should block merging this PR.